submissions/no pain no gain/no pain no gain.md

```diff
@@ -0,0 +1,78 @@
+
+Factorio had a lot of annoying mechanics, that every player hated. I don't know if many of you remember killing biter bases to get some [Alien artifact](https://wiki.factorio.com/Alien_artifact), because that feature was deleted in 0.15 (24.04.2017). For those who don't remember, alien artifacts was used to craft advance materials like modules tier 3 or power armors and also you could craft [purple science](https://wiki.factorio.com/Alien_science_pack), that was the highest tier of science packs in Factorio version 0.15. Constant expansion for more purple orbs wasn't in "Factorio style" where idea is to automate everything, so they got removed and very well.
+
+![](https://i.imgur.com/3JcC4lx.gif)
+
+In Minecraft you need iron pickaxe to dig diamonds, but you need stone and wood first. Factorio was inspired by this idea and also had 4 difference types of pickaxes (wooden and stone was deleted in ancient times) used to mine better "tiers" of resources, but this type of gathering resources in early game was just a distraction from core gameplay.
+
+![](https://i.imgur.com/UieIsz3.gif)
+
+However iron and steel axes was left in the game for a long time and both tools provided difference problems.
+[Iron axe](https://wiki.factorio.com/Iron_axe) was an unnecessary step towards your factory, crafted only once or twice to provide you first iron. What's more new players had problems to find, craft or use this tool what created difficulties and frustration in first few minutes with game.
+[Steel axe](https://wiki.factorio.com/Steel_axe) on the other hand was with us throughout the whole game, crafted from time to time in a bulk amount.
+
+![](https://i.imgur.com/jtQxq0U.gif)
+
+There is more things like: old toolbar, old blueprints management, not upgrade planner, old oil processing, fluids system, no auto replacing belts or homing biters shots. All of this and many more was removed/upgraded during factorio early access. Many games in alpha/beta stage gets a lot of new buildings, mechanics or enemies, but factorio chose difference path and instead of adding more they improved what is already there. People who feel lack of content in factorio can always finds something in huge [library of mods](https://mods.factorio.com/), because stable core makes modding easier.
+
+If you want to read more about removed mechanics there is old fff [Cleanup of mechanics](https://factorio.com/blog/post/fff-266), that will explain in detail decisions behind: pickaxes, furnace heating, burner efficiency and ore hardness.
+
+
+### All your factorio pain in one video...
+You might already saw this video, because it was quite popular on reddit and now is going viral on YouTube, but if not then this video collected (almost) all painful things from factorio like <s>xterminator</s> being killed by a train or looking at your stuck spidertrone, into fast paced compilation.
```

Comment:
Due to the delay in the release, this section needs rewording to past tense. Maybe mention the part 2 video instead?